src/lib/jsonParseV2/parseAttributeValue.cpp

```diff
@@ -50,7 +50,7 @@ std::string parseAttributeValue(ConnectionInfo* ciP, ContextAttribute* caP)
 
   if (document.HasParseError())
   {
-    alarmMgr.badInput(clientIp, "JSON parse error");
+    alarmMgr.badInput(clientIp, "JSON parse error", "");
     oe.fill(SccBadRequest, ERROR_DESC_PARSE, ERROR_PARSE);
     ciP->httpStatusCode = SccBadRequest;
     return oe.toJson();
```

Comment:
Makes sense to return additional information provided by [`document.GetParseError`](https://rapidjson.org/classrapidjson_1_1_generic_document.html#a7607bb42b51547e44bfd4cab35d8f20e) (see the [enum](https://rapidjson.org/group___r_a_p_i_d_j_s_o_n___e_r_r_o_r_s.html#ga7d3acf640886b1f2552dc8c4cd6dea60))?

Reply:
Ok

Reply:
Fixed in 10cdfce2c